src/pages/Dashboard/RewardsDashboard.tsx

```diff
@@ -3,15 +3,15 @@ import { Button, Flex, Heading } from '@chakra-ui/react'
 import { useTranslate } from 'react-polyglot'
 import { Link as NavLink } from 'react-router-dom'
 import { Main } from 'components/Layout/Main'
-import { DeFiEarn } from 'components/StakingVaults/DeFiEarn'
+import { WalletDefiEarn } from 'components/StakingVaults/WalletDefiEarn'
```

Comment:
Do we want to keep the original naming here?

Reply:
We are going to keep this for a bit and rename later if need be.